src/GitTfs/Core/TfsChangeset.cs

```diff
@@ -185,12 +185,14 @@ private LogEntry MakeNewLogEntry(IChangeset changesetToLog, IGitTfsRemote remote
             catch
             {
             }
+            bool foundInAuthorFile = false;
             var name = changesetToLog.Committer;
             var email = changesetToLog.Committer;
-            if (_authors != null && _authors.Authors.ContainsKey(changesetToLog.Committer))
+            if (_authors?.Authors?.ContainsKey(changesetToLog.Committer) ?? false)
             {
                 name = _authors.Authors[changesetToLog.Committer].Name;
                 email = _authors.Authors[changesetToLog.Committer].Email;
+                foundInAuthorFile = true;
             }
             else if (identity != null)
             {
```

Comment:
Sorry, I see know how my suggestion was easy to missunderstand. I meant something like this:

```csharp
var name = changesetToLog.Committer;
var email = changesetToLog.Committer;
bool foundInAuthorFile =_authors?.Authors?.ContainsKey(changesetToLog.Committer) ?? false;
if (foundInAuthorFile )
{
    name = _authors.Authors[changesetToLog.Committer].Name;
    email = _authors.Authors[changesetToLog.Committer].Email;
}
```

Reply:
With or without that change, I am fine with it, just wanted to toss around the idea and check if it makes sense to you.

Reply:
Now I got this ^^